scripts/velodyne_16.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+
+###############################################################################
+# Copyright 2018 The Apollo Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###############################################################################
+
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+cd "${DIR}/.."
+
+source "${DIR}/apollo_base.sh"
+
+# TODO(all): This is just an initial commit. Velodyne 64 and 16 share lots of
+# things. We need to select their processes precisely in 'pgrep', 'pkill' and
+# the monitor module.
+function start() {
+    LOG="${APOLLO_ROOT_DIR}/data/log/velodyne16.out"
+    CMD="roslaunch velodyne start_velodyne_16.launch"
+    NUM_PROCESSES="$(pgrep -c -f "velodyne_nodelet_manager")"
+    if [ "${NUM_PROCESSES}" -eq 0 ]; then
```

Comment:
Not sure if this works, did 16 and 64 use the same "velodyne_nodelet_manger" as process name?

Reply:
That's why I add a "TODO" comment there. Everything should work, except the "pgrep, pkill" and monitor module may get confused a bit.